doc/tagbar.txt

```diff
@@ -103,6 +103,10 @@ The following features are supported by Tagbar:
     Fortran, HTML, Java, JavaScript, Lisp, Lua, Make, MatLab, OCaml, Pascal,
     Perl, PHP, Python, REXX, Ruby, Scheme, Shell script, SLang, SML, SQL, Tcl,
     Tex, Vera, Verilog, VHDL, Vim and YACC.
+  - Additional languages are supported through universal-ctags, including
+    CUDA, R, Rust, Go, and many others. See 
+  https://github.com/universal-ctags/ctags/blob/master/docs/news.rst#new-parsers
```

Comment:
```suggestion
    https://github.com/universal-ctags/ctags/blob/master/docs/news.rst#new-parsers
```

Just making sure the indentation level is the same for all content in the bullet point.